src/components/application_manager/include/application_manager/hmi_state.h

```diff
@@ -200,7 +198,7 @@ class HmiState {
   }
 
  protected:
-  utils::SharedPtr<Application> app_;
+  uint32_t app_id_;
```

Comment:
@shoamano83 according to my comment above, propose to rename to `hmi_app_id_`

Reply:
Updated with commit 14270c9b11d5ac824f1d615d8b2f36277d573b53.